Consequences/Geography/BoundingBox.cs

```diff
@@ -0,0 +1,33 @@
+﻿namespace USACE.HEC.Geography;
+public class BoundingBox
+{
+  private Location _upperLeft;
+  private Location _lowerRight;
```

Comment:
I  think some nice Syntactic Sugar on these would be to use Auto-Properties. Instead of greating a seperate method getter and private backing variable, do Public Location UpperLeft {get;}. Which does the same thing, but lets you interact with it like a simple field. Only supplying the getter means it can only be set in the constructor, which seems reasonable to me.